### ISSUE TITLE
problem on  getting the decision variable in different domain?

### ISSUE THREAD

**Issue by tao0708** [chocoteam/choco-solver#591]
Hi,
     I have a problem: the decision variable in one big domain, and then as the environment changes, divide the big domain into 3 small different domain, then get the decision variable in different domain.
     Any tutorial about this?

**Comment by cprudhom** [comment]
Hi,

Could you rephrase your question and exemplify it ?

Best,
CP

**Comment by tao0708** [comment]
Okay, the range of decision variable is changed.
there is a example:
F0(t)=[i|Ai>t,i=1,2,3,4], F1(t) = [i|Ai <=t];
int[] Ai={29,38,123,56}; int t =0;

first iteration: t=29, get the value of F0 and F1,the the decision variable is in domain F1; 
if F0(t)>=1, then second iteration: t = 38, get the value of F0 and F1, the decision variable is in domain F1.
if F0(t)=0, the program is end.

Hope I have make it clearly.
